pom.xml

```diff
@@ -238,6 +238,7 @@
                     <encoding>UTF-8</encoding>
                     <consoleOutput>true</consoleOutput>
                     <failsOnError>true</failsOnError>
+                    <resourceIncludes>**/*.properties,**/*.jelly,**/*.html</resourceIncludes>
```

Comment:
will jelly files be treated like ordinary html files? or any other standard for jelly files?